### ISSUE TITLE
Fix intermittent connection on Sheila's lid interlock

### ISSUE THREAD

**Issue by arthurrowland** [DoESLiverpool/somebody-should#1563]
Reporting on behalf of someone whose name I didn't catch using Sheila today.

As I understood it:

- Laser worked well for most of day (~5 working hours, don't know how much machine-time)
- Laser head moves but does not fire
- Turning off and on again did not resolve

I don't currently have time to diagnose and fix. Could be interlock issue?

**Comment by Robotorium** [comment]
Sometimes Sheila's door sensor moves out of true and she doesn't detect
that the door is shut. I completely forgot about that. It's on the right
hand side when you open the door.

Op wo 24 mrt. 2021 14:24 schreef arthurrowland ***@***.***>:

**Comment by Robotorium** [comment]
Or the lid, rather.

Op wo 24 mrt. 2021 14:26 schreef Snoof Hattersley ***@***.***>:

**Comment by arthurrowland** [comment]
Thanks Snoof, I just did a quick test and the beam does fire. I didn't load a file but seems very likely it will work. And before I'd submitted this comment the person mentioned in first post went back to using it after a stint on Gerald.

On a related note, not one I reported at the time, Sheila did stop firing in the middle of a cut a couple of weeks ago. I assumed this was due to a large file (complex geometry) and the CNC code was truncated at some point. If I recall right (unreliable narrator), it stopped moving and firing at the same time. Might do a deliberately huge file at some point to test this hypothesis, unless anyone else has figured it out already :)

**Comment by Robotorium** [comment]
I don't think that's ever happened to me, but knowing you the file is
probably bigger and more complex than anything I've ever made 😂 I'll see
if Chris has any ideas when he gets out of work

Op wo 24 mrt. 2021 14:47 schreef arthurrowland ***@***.***>:

**Comment by arthurrowland** [comment]
I think we're OK on this issue so I'll just leave you with this photo by way of a partial explanation for the large file size :P

![image](https://user-images.githubusercontent.com/78351702/112331218-a24cb180-8cb0-11eb-9201-2911469b9b73.png)
image description: photograph of wide bracelet worn on Arthur's wrist, made from plywood laser cut with geometric pattern to allow it to be very flexible

**Comment by Robotorium** [comment]
Reopening this - she's cut stuff like that before with no issues. She was doing the same again quite a lot today, may need looking at. I noticed, when fiddling with the sensor that detects whether the door is shut, that the relevant wires hang down over a part that gets reasonably warm, so I'm not sure if that's related somehow.

**Comment by JackiePease** [comment]
This happened a couple of times today while Liz Taylor was cutting.

**Comment by arthurrowland** [comment]
Specified the issue as I'm pretty confident it was the interlock.

I noticed there was a bolt missing and that it was possible to rotate the sensor around the other screw:

![sensor in the laser cutter's frame with one bolt and one empty hole](https://user-images.githubusercontent.com/78351702/120307765-45323500-c2cb-11eb-887d-c51b95b750fa.png)

My theory is that this is the cause of power cutting out - the screw in the lid failing to make reliable contact since the sensor is lower than it should be.

So I have refitted it with two tight bolts and tightened both (sorry for using a different bolt head - promise I'm not trying to wind people up with all this disorder):

![sensor in laser cutter with two bolts in](https://user-images.githubusercontent.com/78351702/120307892-6561f400-c2cb-11eb-913b-2b3485097abd.png)

I ran a brief test to check it fired - worked for me.

Could Sheila users keep tabs on this and report back if it happens again?

If we don't hear of problems by the end of June, I move to close this issue.
